paper/paper.bib

```diff
@@ -0,0 +1,64 @@
+
+@article{hunter_matplotlib:_2007,
+	title = {Matplotlib: {A} 2D graphics environment},
+	volume = {9},
+	shorttitle = {Matplotlib},
+	number = {3},
+	journal = {Computing in science \& engineering},
+	author = {Hunter, John D.},
+	year = {2007},
+	pages = {90},
+	file = {Snapshot:/Users/madicken/Zotero/storage/5IJM9CLA/MCSE.2007.html:text/html}
+}
+
+@article{turk_yt:_2010,
+	title = {yt: {A} multi-code analysis toolkit for astrophysical simulation data},
+	volume = {192},
+	shorttitle = {yt},
+	number = {1},
+	journal = {The Astrophysical Journal Supplement Series},
```

Comment:
I *think* the publication date is actually 2011, weirdly.

Reply:
Yep! It looks like Zotero scraped the draft date not the publication date.